supervisor/hyperpod.go

```diff
@@ -366,7 +366,12 @@ func (hp *HyperPod) startNsListener() (err error) {
 
 func (hp *HyperPod) stopNsListener() {
 	if hp.nslistener != nil {
-		hp.nslistener.cmd.Process.Kill()
+		go func() {
+			hp.nslistener.cmd.Process.Kill()
+		}()
+		if err := hp.nslistener.cmd.Process.Signal(syscall.Signal(0)); err == nil {
+			hp.nslistener.cmd.Wait()
```

Comment:
The signal(0) was just to ensure the process is really alive to prevent runtime panic on cmd.Wait.
Do you prefer it like this?
```
-               hp.nslistener.cmd.Process.Kill()
+               go func() {
+                       time.Sleep(time.Second*1)
+                       hp.nslistener.cmd.Process.Kill()
+               }()
+               hp.nslistener.cmd.Wait()
```